src/utils/clearPeopleList.ts

```diff
@@ -0,0 +1,27 @@
+import { Person } from '../types';
+import { sortPeople } from './sortPeople';
+
+export function clearPeopleList(
+  people: Person[],
+  centuries: string[],
+  sex: string | null,
+  query: string | null,
+  sort: string | null,
+  order: string | null,
+) {
+  const visiblePeople = people.filter(
+    person =>
+      (!sex || person.sex === sex) &&
+      (person.name.includes(query || '') ||
+        person.motherName?.includes(query || '') ||
+        person.fatherName?.includes(query || '')) &&
+      (!centuries.length ||
+        centuries.includes(Math.ceil(person.born / 100).toString())),
```

Comment:
The filtering logic for centuries converts the birth year to a century using `Math.ceil(person.born / 100)`. Ensure that the `centuries` array contains string representations of centuries, as the `includes` method is checking against strings.